index.test-d.ts

```diff
@@ -0,0 +1,48 @@
+import { expectType } from 'tsd-check';
+import { PassThrough } from 'stream';
+import ora, { promise } from '.';
+
+const spinner = ora('Loading unicorns');
+ora({ text: 'Loading unicorns' });
+ora({ spinner: 'squish' });
+ora({ spinner: { frames: ['-', '+', '-'] } });
+ora({ spinner: { interval: 80, frames: ['-', '+', '-'] } });
+ora({ color: 'cyan' });
+ora({ hideCursor: true });
+ora({ indent: 1 });
+ora({ interval: 80 });
+ora({ stream: new PassThrough() });
+ora({ isEnabled: true });
```

Comment:
```suggestion
ora({isEnabled: true});
```

For all braces.

Reply:
Is it allowed to add a Prettier config to this project? My default settings conflict with this and I keep forgetting editing this back after pretty-printing.

Reply:
No, you should not have Prettier enabled when there's no prettier config in a project.